examples/ui/text_debug.rs

```diff
@@ -106,124 +106,121 @@ fn infotext_system(mut commands: Commands, asset_server: Res<AssetServer>) {
-                    font: font.clone(),
-                    font_size: 29.0,
-                    color: YELLOW.into(),
-                },
-            )
-            .with_text_justify(JustifyText::Left)
-            .with_style(Style {
+        builder.spawn((TextNEW::new(
+            "This text is left-justified and is vertically positioned to distribute the empty space equally above and below it."),
+            TextStyle {
+                font: font.clone(),
+                font_size: 29.0,
+                color: YELLOW.into(),
+            },
+            TextBlock::new_with_justify(JustifyText::Left),
+            Style {
                 max_width: Val::Px(300.),
                 ..default()
-            }),
+            })
         );
 
-        builder.spawn(
-            TextBundle::from_section(
-                "This text is fully justified and is positioned in the same way.",
-                TextStyle {
-                    font: font.clone(),
-                    font_size: 29.0,
-                    color: GREEN_YELLOW.into(),
-                },
-            )
-            .with_text_justify(JustifyText::Justified)
-            .with_style(Style {
+        builder.spawn((
+            TextNEW::new("This text is fully justified and is positioned in the same way."),
+            TextStyle {
+                font: font.clone(),
+                font_size: 29.0,
+                color: GREEN_YELLOW.into(),
+            },
+            TextBlock::new_with_justify(JustifyText::Justified),
+            Style {
                 max_width: Val::Px(300.),
                 ..default()
-            }),
+            })
         );
 
         builder.spawn((
-            TextBundle::from_sections([
-                TextSection::new(
-                    "",
-                    TextStyle {
-                        font: font.clone(),
-                        font_size: 21.0,
-                        ..default()
-                    },
-                ),
-                TextSection::new(
-                    "\nThis text changes in the bottom right",
-                    TextStyle {
-                        font: font.clone(),
-                        font_size: 21.0,
-                        ..default()
-                    },
-                ),
-                TextSection::new(
-                    " this text has zero fontsize",
-                    TextStyle {
-                        font: font.clone(),
-                        font_size: 0.0,
-                        color: BLUE.into(),
-                    },
-                ),
-                TextSection::new(
-                    "\nThis text changes in the bottom right - ",
-                    TextStyle {
-                        font: font.clone(),
-                        font_size: 21.0,
-                        color: RED.into(),
-                    },
-                ),
-                TextSection::from_style(TextStyle {
+            TextNEW::new(""),
```

Comment:
There really needs to be a text node constructor function that takes an iterator of strings as this stuff is torturous.
It's especially annoying if you have a list of items when you need to split off the head of the list and add it with `TextNEW`.